tests/compute/test_sampling.py

```diff
@@ -96,22 +99,91 @@ def test_random_walk():
         g4, [0, 1, 2, 0, 1, 2], metapath=metapath, return_eids=True)
     check_random_walk(g4, metapath, traces, ntypes, trace_eids=eids)
 
-    metapath = ['follow', 'view', 'viewed-by'] * 2
+    # Non-uniform prob is not supported in GPU.
+    if F._default_context_str != 'gpu':
```

Comment:
Could we split this to a separate unit test, such that in the log, we can see that non-uniform prob isn't being run on the GPU?

Reply:
Split the random walk test to `test_non_uniform_random_walk()` (for CPU only) and `test_uniform_random_walk(use_uva)` with parametrized CPU(use_uva=False) and GPU(use_uva=[True, False]).